apps/sms/test/marionette/lib/messages.js

```diff
@@ -2,9 +2,13 @@
 
 /* global module */
 var InboxAccessor = require('./views/inbox/accessors');
-var ComposerAccessor = require('./views/new-message/accessors');
+var NewMessageAccessor = require('./views/new-message/accessors');
 var NewMessageView = require('./views/new-message/view');
 var ConversationAccessor = require('./views/conversation/accessors');
+var MenuAccessor = require('./views/shared/menu_accessors');
```

Comment:
All these accessors and duplication here is for transitional period until we migrate all tests to new view-based-paradigm